scripts/SetupDeployedTransmuter.s.sol

```diff
@@ -0,0 +1,209 @@
+            xBurnFeeIB01[0] = uint64(BASE_9);
+            xBurnFeeIB01[1] = uint64((16 * BASE_9) / 100);
+            xBurnFeeIB01[2] = uint64((15 * BASE_9) / 100);
+
+            int64[] memory yBurnFeeIB01 = new int64[](3);
+            yBurnFeeIB01[0] = int64(uint64((50 * BASE_9) / 10000));
+            yBurnFeeIB01[1] = int64(uint64((50 * BASE_9) / 10000));
+            yBurnFeeIB01[2] = int64(uint64(MAX_BURN_FEE));
+
+            bytes memory oracleConfig;
+            {
+                bytes memory readData;
+                {
+                    AggregatorV3Interface[] memory circuitChainlink = new AggregatorV3Interface[](1);
+                    uint32[] memory stalePeriods = new uint32[](1);
+                    uint8[] memory circuitChainIsMultiplied = new uint8[](1);
+                    uint8[] memory chainlinkDecimals = new uint8[](1);
+
+                    // Chainlink IB01/USD oracle
+                    circuitChainlink[0] = AggregatorV3Interface(0x32d1463EB53b73C095625719Afa544D5426354cB);
+                    stalePeriods[0] = 1 days;
+                    circuitChainIsMultiplied[0] = 1;
+                    chainlinkDecimals[0] = 8;
+                    Storage.OracleQuoteType quoteType = Storage.OracleQuoteType.UNIT;
+                    readData = abi.encode(
+                        circuitChainlink,
+                        stalePeriods,
+                        circuitChainIsMultiplied,
+                        chainlinkDecimals,
+                        quoteType
+                    );
+                }
+
+                // Current value is 109.43, but we need to update it now otherwise we'll have to wait for a week
+                // before we can update it
+                bytes memory targetData = abi.encode(
+                    109500000000000000000,
+                    uint96(DEVIATION_THRESHOLD_IB01),
+                    uint96(block.timestamp),
+                    HEARTBEAT
+                );
+
+                oracleConfig = abi.encode(
+                    Storage.OracleReadType.CHAINLINK_FEEDS,
+                    Storage.OracleReadType.MAX,
+                    readData,
+                    targetData,
+                    abi.encode(uint80(0), uint80(50 * BPS), uint80(50 * BPS))
+                );
+            }
+            collaterals[0] = CollateralSetupProd(
+                BIB01,
+                oracleConfig,
+                xMintFeeIB01,
+                yMintFeeIB01,
+                xBurnFeeIB01,
+                yBurnFeeIB01
+            );
+        }
+
+        // steakUSDC -> max oracle or target oracle
+        {
+            uint64[] memory xMintFeeSteak = new uint64[](3);
+            xMintFeeSteak[0] = uint64(0);
+            xMintFeeSteak[1] = uint64((79 * BASE_9) / 100);
+            xMintFeeSteak[2] = uint64((80 * BASE_9) / 100);
+
+            int64[] memory yMintFeeSteak = new int64[](3);
+            yMintFeeSteak[0] = int64(0);
```

Comment:
Let's set a fee on this one as we don't need to have eople coming in with steakUSDC as we can easily rebalance for no cost + protection put inplace will make it hard to be not trickable